R/shiny.R

```diff
@@ -297,23 +297,135 @@ sleuth_live <- function(obj, settings = sleuth_live_settings(),
               choices = c(NULL, poss_covars), selected = NULL)
             ),
           column(2,
-            numericInput('pca_point_size', label = 'size: ', value = 3))
-          ),
-        fluidRow(
+            numericInput('pca_point_size', label = 'size: ', value = 3)),
           column(2,
             selectInput('pca_units', label = 'units: ',
               choices = c('est_counts', 'tpm'),
-              selected = 'est_counts')),
-          column(3,
+              selected = 'est_counts'))
+        ),
+        fluidRow(
+          column(2,
             checkboxInput('pca_filt', label = 'filter',
-              value = TRUE),
+              value = TRUE)
+              ),
+          column(2,
             checkboxInput('text_labels', label = 'text labels',
               value = TRUE)
             )
           ),
-        fluidRow(plotOutput('pca_plt'))
+
+        fluidRow(plotOutput('pca_plt')),
+
+        fluidRow(
+        column(12,
+          p(h3('loadings'), "observe contributions of samples or transcripts to the principal component")
+          ),
+          offset = 1),
+        fluidRow(
+          column(3,
+            textInput('sample', label = 'transcript: ', value = '',
+              )
+            ),
+          column(2,
+            selectInput('PC', label = 'PC: ', choices = 1:5,
+              selected = 1)
+            ),
+          column(3,
+            selectInput('pc_count', label = '# of PCs or transcripts: ', choices = 1:10,
+              selected = 5)),
+          column(2,
+            checkboxInput('absl', label = 'absolute value',
+              value = TRUE)
+            ),
+          column(2,
+            checkboxInput('bool', label = 'scale',
+              value = FALSE)
+            )
+          ),
+
+        fluidRow(
+          column(12,
+            p(h3('variance explained'))
+            ),
+            offset = 1),
+        fluidRow(plotOutput('plt_pc_var')),
+        fluidRow(
+          column(12,
+            p(h3('loadings'))
+            ),
+            offset = 1),
+        fluidRow(plotOutput('plt_pc_loadings'))
         ),
 
+      ####
```

Comment:
let's try not to leave large amounts of commented code around. I don't think this code is particularly necessary anymore?